rocksdb_admin/tests/admin_handler_test.cpp

```diff
@@ -882,6 +882,23 @@ TEST(AdminHandlerTest, MetaData) {
   EXPECT_EQ(meta.db_name, db_name);
 }
 
+
+TEST(AdminHandlerTest, InitS3Tmp) {
+  gflags::FlagSaver saver;
+  FLAGS_rocksdb_dir = "/tmp/admin_handler_test/";
+  auto s3_path = FLAGS_rocksdb_dir + "s3_tmp/"; 
+  auto test_directory = s3_path + "test_directory";
+  clearAndCreateDir(FLAGS_rocksdb_dir);
```

Comment:
didn't realize we have `clearAndCreateDir` already, should use move that into a production utilization function, and use that in the s3_tmp directory initialization?

Reply:
wdyt?